adhocteam/Head-Start-TTADP
# CD: Fix migration, clean up and explain env variables

**Description of change**
Unfortunately, https://github.com/adhocteam/Head-Start-TTADP/pull/90 didn't work quite as I expected. Because I had first added the `POSTGRES_*` variables directly to the sandbox application in cloud.gov I didn't realize my variable writing didn't work. This corrects the variable writing.

Also, it cleans up other environment variables that are passed to the application and adds step by step instructions for developers to add their own env variables to the application.

**How to test**
Delete all postgres related vars from the sandbox application (via cloud.gov ui). Rerun workflow in circleci. See that sandbox still works and migrations ran (check out the cloud.gov env vars and the log stream).

**Issue(s)**
* https://github.com/HHS/Head-Start-TTADP/issues/202

**Checklist**

- [n/a] Meets accessibility standards (WCAG 2.1 Levels A, AA)
    - API methods
    - Boundary and Data Flow Diagrams
    - [Architectural Decision Records](https://adr.github.io/) written for major infrastructure decisions with the [Nygard template](https://github.com/joelparkerhenderson/architecture_decision_record/blob/master/adr_template_by_michael_nygard.md)
    - OSCAL templates completed when security controls are implemented or modified
